backend/uasmartsignage/src/main/java/deti/uas/uasmartsignage/Models/Schedule.java

```diff
@@ -22,17 +23,16 @@ public class Schedule {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    private String frequency;
-
-    private int nTimes;
+    @Column(name = "frequency")
+    private int frequency;
 
     private int intervalOfTime;
```

Comment:
Unsure what this does.

Reply:
The attribute "intervalOfTime" was erased because it is not needed.